poe-tasks/build-and-publish-java-connectors-with-tag.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Usage examples:
+#   ./build-and-publish-java-connectors-with-tag.sh destination-dev-null source-bigquery
+#   ./get-modified-connectors.sh --prev-commit | ./build-and-publish-java-connectors-with-tag.sh
+set -euo pipefail
+
+CONNECTORS_DIR="airbyte-integrations/connectors"
+
+# ---------- helper: collect connector names ----------
+collect_connectors() {
+  if [ "$#" -gt 0 ]; then
+    # names were passed as CLI args
+    printf '%s\n' "$@"
+  else
+    # expect JSON on stdin from previous script
+    if [ -t 0 ]; then
+      echo "Error: no connector names given and no JSON piped to stdin." >&2
+      exit 1
+    fi
+    jq -r '.connector[]'
+  fi
+}
+
+# ---------- main loop ----------
+build_connector() {
+  local connector="$1"
+  local meta_file="${CONNECTORS_DIR}/${connector}/metadata.yaml"
+
+  if [[ ! -f "$meta_file" ]]; then
+    echo "metadata.yaml not found for ${connector}" >&2
+    exit 1
+  fi
+
+  local image_tag
+  image_tag=$(yq -r '.data.dockerImageTag' "$meta_file")
```

Comment:
Do you know if `yq` is installed where we need to run it? I know we take `jq` for granted but I don't know if we can do the same for `yq`.

Reply:
Good question & thanks for checking. Also happy to refactor. Mostly trying to get things working instead of perfect.